src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -97,6 +98,103 @@ struct ComputeNormalDotFluxes {
                         interface_unit_normal) noexcept;
 };
 
+/*!
+ * \ingroup NumericalFluxesGroup
+ * \brief Computes the penalty flux
+ *
+ * The penalty flux is given by:
+ * \f{align}
+ * F^*(\Psi) =& F(\Psi) = 0 \\
```

Comment:
this is `F^*`, the flux form for inter-element boundaries

Reply:
That's `G`, there's no `F^*`

Reply:
Thats confusing .. what are the `F^*`  in [here](https://spectre-code.org/structScalarWave_1_1UpwindFlux.html)?

Reply:
It's also `G`. They should all be changed to `G`. `F^*` isn't a thing. I'm not actually sure where that came from but it's not a notation I've ever seen outside of some of the spectre dox.

Reply:
(Hesthaven's book, at the very least, uses this notation)

Reply:
Hesthaven's [this](https://www.cambridge.org/core/books/spectral-methods-for-timedependent-problems/FF1D2BA7F74E5A8930F1F674CC80511A) book, and [this](https://link.springer.com/book/10.1007/978-0-387-72067-8) book, and Saul's [paper](https://www.sciencedirect.com/science/article/pii/S0021999116000954?via%3Dihub) - all use `F^*`.

Reply:
Except none of those are correct and use the wrong normal... We've settled on `G` for this, which is what is used in the ADER-DG lit

Reply:
Sure, I'll go ahead and change to `G`.

Reply:
Sorry, I edited my comment. The thing I've always seen is `F^{*i}n_i` as the correction, which is incorrect because it assumes the normal on the two sides are the same. `G` is used to denote the correct boundary term that is motivated by a numerical flux but accounts for the possibility of different normal vectors :)